src/thing_navigate.c

```diff
@@ -47,6 +47,16 @@ DLLIMPORT long _DK_get_next_gap_creature_can_fit_in_below_point(struct Thing *cr
 }
 #endif
 /******************************************************************************/
+
+// Call this function if you don't want the creature/thing to (visually) fly across the map whenever suddenly moving a very far distance. (teleporting for example)
```

Comment:
This comment won't show up when you hover the function. There's a way to format comments that the information is more easily available, see:
![image](https://user-images.githubusercontent.com/13840686/185004351-39054326-bebd-4808-94b1-23942f0de8ec.png)